### PR TITLE
Do not use shell syntax to get the first remote

### PR DESCRIPTION
**Pull request by linquize**
This is to make it work on Windows.

### CODE REVIEW

File: bin/git-stats-importer [hunk 1]
```diff
@@ -33,82 +33,93 @@ var myRepo = new Repo(repoDir);
...
-
-                    if (!/^commit\ /.test(cLine)) { return doSeq(i + 1); }
-                    function getDate(x) {
-                        var date = (data[x].match(/^Date: (.*)/) || [])[1];
-                        if (date) { return date.trim(); }
-                        return getDate(x + 1);
-                    }
+        myRepo.exec(OArgv({ get: "remote." + REMOTE + ".url" }, "config"), function (err, REMOTE_URL) {
+            if (!REMOTE_URL) {
+                return Logger.log("Cannot find the remote origin url. Please add it.", "error");
```
Comment (IonicaBizau) [inline bin/git-stats-importer]: We have to update the error message: *Cannot find the <REMOTE> url.*, or something like that (maybe you have better ideas).

Thanks for this PR! :+1:

Comment (linquize) [inline bin/git-stats-importer]: Changed

Comment (IonicaBizau) [inline bin/git-stats-importer]: :+1: Cool! Can you please add yourself in`contributors`?

Comment (linquize) [inline bin/git-stats-importer]: Added

**Comment by IonicaBizau** [comment]
Thanks! :+1:

**Pull request merged**